### ISSUE TITLE
Hall_symbol in phonopy_disp.yaml can be an invalid YAML string

### ISSUE THREAD

**Issue by ExpHP** [atztogo/phonopy#69]
Double quotes can appear in Hall symbols, producing an invalid `phonopy_disp.yaml` file:

```yaml
space_group:                                                                                        
  type: "P-3m1"                                                                                     
  number: 164                                                                                       
  Hall_symbol: "-P 3 2""
```

Sample input: [phonopy-yaml-test-files.tar.gz](https://github.com/atztogo/phonopy/files/2068885/phonopy-yaml-test-files.tar.gz)

**Comment by atztogo** [comment]
Thanks. I fixed it at 54d0f2812ee736e2ab65f732d346e316e86319e4.

**Comment by ExpHP** [comment]
Thanks!
